hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -919,13 +951,12 @@ public HapiSpec creationRequiresAppropriateSigsHappyPath() {
                         .hasKnownStatus(SUCCESS));
     }
 
+    @HapiTest
     public HapiSpec creationValidatesTreasuryAccount() {
         return defaultHapiSpec("CreationValidatesTreasuryAccount")
                 .given(cryptoCreate(TOKEN_TREASURY).balance(0L))
                 .when(cryptoDelete(TOKEN_TREASURY))
-                .then(tokenCreate("shouldntWork")
-                        .treasury(TOKEN_TREASURY)
-                        .hasKnownStatus(INVALID_TREASURY_ACCOUNT_FOR_TOKEN));
+                .then(tokenCreate("shouldntWork").treasury(TOKEN_TREASURY).hasKnownStatus(ACCOUNT_DELETED));
```

Comment:
I think we should try to preserve the old behavior.

Reply:
+1